src/test/locale/es-do.js

```diff
@@ -211,3 +211,9 @@ test('weeks year starting sunday formatted', function (assert) {
     assert.equal(moment([2012, 0, 15]).format('w ww wo'),   '2 02 2º', 'Jan 15 2012 should be week 2');
 });
 
+test('test short months proper', function (assert) {
+    var str = moment(new Date(2016, 7, 2)).format('DD-MMM-YYYY'); // "02-ago-2016"
```

Comment:
Is this code necessary? Why not just do `str=02-ago-2016`? That's what it will always be.